reference/constraints/Hostname.rst

```diff
@@ -0,0 +1,135 @@
+Hostname
+========
+
+This constraint ensures that the given value is a valid and non-reserved
+host name (internally it uses the ``FILTER_VALIDATE_DOMAIN`` option of the
+:phpfunction:`filter_var` PHP function).
+
+.. versionadded:: 5.1
+
+    The ``Hostname`` constraint was introduced in Symfony 5.1.
+
+==========  ===================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Options     - `groups`_
+            - `message`_
+            - `payload`_
+            - `requireTld`_
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\Hostname`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\HostnameValidator`
+==========  ===================================================================
```

Comment:
Underline too short

Reply:
YEs ... but RST allows that for tables ... and we deicded to use 80-character long tables if possible (the second table of this article uses 80-character tables too, although it really needs much less characters). Is to avoid reformatting things for every change in the tables.